test-forms/index.json

```diff
@@ -120,7 +120,7 @@
   { "html": "samash_signup.html" },
   { "html": "financialtimes_login.html", "title": "Login" },
   { "html": "containerstore_login_signup.html" },
-  { "html": "containerstore_forgot_password.html", "expectedFailures": ["username"] },
+  { "html": "containerstore_forgot_password.html", "expectedFailures": ["password.new"] },
```

Comment:
So here we are breaking the password field now, but on the other hand we're fixing the username field. I think its a net win, given the site in question (sharperimages.com).

Reply:
This is unfortunate. I've spent 30 mins looking for a quick win but couldn't find anything obvious.

Note that containerstore has a higher domain rank than what we're fixing now, but in this case we're just changing what breaks on containerstore, so it's not a net regression. FYI, you can check domain rank using https://tranco-list.eu/query.